content/post/My-journey-with-ALiAS.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "My journey with ALiAS : From a commit to the community"
+slug = "FOSS communities"
+image = "/images/community.jpg"
+date = "2019-05-25T00:00:00"
+tags='ALiAS, FOSS'
```

Comment:
```suggestion
tags=['ALiAS', 'FOSS', 'Experiences', 'memories', `Alumni`]
```

Reply:
> Good stuff, thanks for writing this blog post. Some changes that are needed and review made inline
> 
> * Run the content through Grammarly for tweaking the tone.

Yeah, apparently I have this bad habit of intentionally omitting articles while writing/typing to save time :P. Gotta fix this.

> * You can add images of your time with ALiAS, meetups, our events, hackathons, prizes stuff that inspires people further.

Yeah, that would take some time to extract pics from social profiles of ALiAS, since I don't have any personal collection of those.

> * Why not write more? This is a great start. What say?
Yeah that's the plan. I'm just getting started.

Reply:
Well, picking one to pictures won't be that hard. 
How about adding some of ILuGD or LinuxChix meetups that you went to.

Reply:
> Well, picking one to pictures won't be that hard.
> How about adding some of ILuGD or LinuxChix meetups that you went to.

I guess PyDelhi or mega meetups pics would better for this, since they had more attendance from ALiAS so those pics would resonate more with my ALiAS journey. What say?

Reply:
Go for it, I saw a picture of a recent Ilugd meetup with you in it. Put that as well.

Reply:
?

Reply:
Done.